### ISSUE TITLE
Cannot create new boards: "Request failed with status code 500"

### ISSUE THREAD

**Issue by HLFH** [nextcloud/deck#1985]
**Describe the bug**
I click on "Create new board", write the name of the board, click Enter or on the arrow.
Nothing happens.

**To Reproduce**
1. Click on 'Create new board'
2. Write the name of the board
3. Click Enter or on the arrow
4. Nothing happens, board does not seem to be created.

**Expected behavior**
Board should be created when we are creating it.

**Screenshots**
<img width="482" alt="Screenshot 2020-06-05 at 12 33 33" src="https://user-images.githubusercontent.com/2359059/83871879-d0183800-a728-11ea-879d-47fc888d70e7.png">

And POST error HTTP 500 on POST apps/deck/boards endpoint.

**Client details:**
 - OS: macOS Catalina, 10.15.5 Beta (19F94a)
 - Browser: Chrome
 - Version: 81.0.4044.138 (Official Build) (64-bit)
 - Device: MacBookAir8,1

**Comment by HLFH** [comment]
In nextcloud.log, this is interesting:

> 
![image](https://user-images.githubusercontent.com/2359059/83873518-de1b8800-a72b-11ea-8812-09603d6f8ff9.png)

**Comment by HLFH** [comment]
Only these columns are present in oc_deck_boards: 
<img width="497" alt="Screenshot 2020-06-05 at 13 19 16" src="https://user-images.githubusercontent.com/2359059/83875552-3607be00-a72f-11ea-9ba3-4f40cefd6246.png">

**Comment by juliushaertl** [comment]
Did you install deck for the first time or was there an old version present on the system at some point before? If so what version was that?

**Comment by HLFH** [comment]
Thanks for the help.

I deleted all oc_deck_* tables.
<img width="1035" alt="Screenshot 2020-06-05 at 13 58 11" src="https://user-images.githubusercontent.com/2359059/83880890-cc3fe200-a737-11ea-8eef-56c526ba1348.png">

I deleted the 'deck' entries in `oc_migrations` table.
<img width="484" alt="Screenshot 2020-06-05 at 14 19 56" src="https://user-images.githubusercontent.com/2359059/83880907-d366f000-a737-11ea-9396-87d58e19839e.png">

This is now working.
The database data related to the `deck` app was probably corrupt.
There was an old version present on the system at some point before, and I don't know which one was it. 

Consider it solved then. Thanks again.
